### ISSUE TITLE
When I install the template "self-query-qdrant" I get this error: UnicodeDecodeError: 'charmap' codec can't decode byte 0x81 in position 132: character maps to <undefined>

### ISSUE THREAD

**Issue by ivan-schinella** [langchain-ai/langchain#14341]
### System Info

Windows 10 Pro - 22H2 - Build 9045.3693 - Windows Feature Experience Pack 1000.19053.1000.0
Python 3.11.5
langchain-cli 0.0.19

### Information

- [ ] The official example notebooks/scripts
- [ ] My own modified scripts

### Related Components

- [ ] LLMs/Chat Models
- [ ] Embedding Models
- [ ] Prompts / Prompt Templates / Prompt Selectors
- [ ] Output Parsers
- [ ] Document Loaders
- [ ] Vector Stores / Retrievers
- [ ] Memory
- [ ] Agents / Agent Executors
- [ ] Tools / Toolkits
- [ ] Chains
- [ ] Callbacks/Tracing
- [ ] Async

### Reproduction

1. creating a virtual environments with python - m venv .venv and activate it;
2. installing langchain-cli with pip install -U "langchain-cli[serve]"
3. launch langchain app new qdrant-app --package self-query-qdrant 

### Expected behavior

As for other templates that I have installed, I expected to find the .py files of the app in the packages directory, but it's empty.
[log.txt](https://github.com/langchain-ai/langchain/files/13580197/log.txt)

**Comment by ivan-schinella** [comment]
@dosu-bot
I can't modify files that aren't copied on my PC, I think. Package installation stops before. 
On the other hands, for example, the package installation of "vertexai-chuck-norris" terminate normally without stops.

**Comment by ivan-schinella** [comment]
@dosu-bot
I applied the hints (I found the app.py in my virtual environment .venv\Lib\site-packages\langchain_cli\namespaces), but the installation stops anyway. 
Initially the installation copy locally (into C:\Users\<username>\AppData\Roaming\langchain\git_repos) all the templates. I found the template "self_query_qdrant" "incriminated"  and I noticed that in file pyproject.toml there is line 4 authors = ["Kacper Łukawski <lukawski.kacper@gmail.com>"] with one uncommon character. After that I change it, saved the file and successfully retry the installation!

I think the team must correct that file and update the repo.
